tests/test_pipelines.py

```diff
@@ -371,6 +386,7 @@ class PipelineCommonTests(unittest.TestCase):
         "translation_en_to_fr",
         "translation_en_to_de",
         "translation_en_to_ro",
+        "text_generation",
```

Comment:
I think that it should be `text-generation` instead of `text_generation`. All other pipelines that have two words building them are kebab cased : `feature-extraction`, `question-answering`, `fill-mask`.

I can understand why the translations are different (snake case), but I think this pipeline should keep to kebab case like the other ones. Wdyt @patrickvonplaten?

Reply:
agree